src/styles/utils/_variables.scss

```
@@ -0,0 +1,13 @@
+$Poppins-font-familly: "Poppins";
+$Open-Sans-font-familly: "Open Sans";
+
+$color-white: #fff;
+$very-dark-blue: #253757;
+$gray: #6c788b;
```

It is a good practice to name color variable in this pattern for readability
```suggestion
$color-gray: #6c788b;
```